packages/harmony/tsconfig.json

```diff
@@ -4,22 +4,24 @@
     "lib": ["ES2020", "dom"],
     "module": "esnext",
     "checkJs": false,
+    "plugins": [{ "name": "typescript-plugin-css-modules" }],
     "allowJs": false,
     "jsx": "react-jsx",
     "resolveJsonModule": true,
     "moduleResolution": "node",
     "typeRoots": [
       "./types",
       "./node_modules/@types",
-      "../../node_modules/@types"
+      "../../../node_modules/@types"
```

Comment:
how come we need the typeRoots for harmony?

Reply:
I was just correcting it while i was in here, but I don't know if it is needed actually. This was mimicking the config from libs

Reply:
oh i see, yeah lets try removing and see what's up